lab_1/main.py

```diff
@@ -3,81 +3,148 @@
 Language detection
 """
 
+import json
 
-def tokenize(text: str) -> list or None:
+
+def tokenize(text: str):
     """
     Splits a text into tokens, converts the tokens into lowercase,
     removes punctuation and other symbols from words
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
-    """
-    pass
+        """
+    if isinstance(text, str):
```

Comment:
let's put checks with None at the beginning
this way the code would be more readable

Reply:
done